src/FlaUI.WebDriver.UITests/TestUtil/FlaUIDriverOptions.cs

```diff
@@ -1,52 +1,48 @@
 ﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Appium;
 
 namespace FlaUI.WebDriver.UITests.TestUtil
 {
-    internal class FlaUIDriverOptions : DriverOptions
+    internal class FlaUIDriverOptions : AppiumOptions
     {
-        public override ICapabilities ToCapabilities()
-        {
-            return GenerateDesiredCapabilities(true);
-        }
-
-        public static FlaUIDriverOptions TestApp() => App(TestApplication.FullPath);
+        public static FlaUIDriverOptions TestApp() => ForApp(TestApplication.FullPath);
 
-        public static DriverOptions RootApp() => App("Root");
+        public static FlaUIDriverOptions RootApp() => ForApp("Root");
 
-        public static FlaUIDriverOptions App(string path)
+        public static FlaUIDriverOptions ForApp(string path)
```

Comment:
Added a `From` prefix to these methods because there is an `App` property in `AppiumOptions` that the `App()` method was hiding.